src/services/api/routes/v1/register.js

```diff
@@ -4,11 +4,14 @@ const fs = require('fs-extra');
 const moment = require('moment');
 const crypto = require('crypto');
 const hcaptcha = require('hcaptcha');
+const bcrypt = require('bcrypt');
 const { PNID } = require('../../../../models/pnid');
 const { NEXAccount } = require('../../../../models/nex-account');
 const database = require('../../../../database');
+const cache = require('../../../../cache');
 const util = require('../../../../util');
-const config = require('../../../../../config');
+const logger = require('../../../../../logger');
+const config = require('../../../../../config.json');
```

Comment:
mayhaps setup a root path alias, you can use typescript compiler for that.
then you can just import with `import foo from '@/logger'` or `import foo from '~/logger'`. (two common alias conventions.

Your choice tho, not really neccesary, just looks way nicer and more readable if you do.
(ps: typescript compiler !== typescript language, you can use normal js files that dont use typescript language with the compiler)

Reply:
actually seems like there is an easier method: https://www.npmjs.com/package/module-alias

Reply:
yes